src/views/studio/studio-project-adder.jsx

```diff
@@ -67,7 +67,7 @@ const StudioProjectAdder = ({intl, onSubmit}) => {
                     className={classNames({'mod-form-error': error})}
                     disabled={submitting}
                     type="text"
-                    placeholder={intl.formatMessage({id: 'studio.addProjectPlaceholder'})}
+                    placeholder="http://scratch.mit.edu/projects/xxxx"
```

Comment:
maybe this detail doesn't matter since it's just a placeholder, but looks like our URLs generally use `https`

Reply:
Good catch! Just fixed that.